Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -485,3 +485,119 @@ public extension TransposedConv2D {
             padding: padding)
     }
 }
+
+/// A 2-D depthwise convolution layer.
+///
+/// This layer creates seperable convolution filters that are convolved with the layer input to produce a
+/// tensor of outputs.
+@frozen
+public struct DepthwiseConv2D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The 4-D convolution kernel for the depthwise convolution.
+    public var kernel: Tensor<Scalar>
```

Comment:
This particular property should be called `filter` for consistency with other conv layers.